manifests/ha/install.yaml

```diff
@@ -2354,6 +2484,7 @@ metadata:
     app.kubernetes.io/name: argocd-redis-ha
     app.kubernetes.io/part-of: argocd
   name: argocd-redis-ha-announce-0
+  namespace: default
```

Comment:
This should not be here. The namespace should not be hard-wired into the yaml.

Reply:
This is being pulled in from the upstream redis-ha chart: https://github.com/helm/charts/commit/e1b39e9cd3c80aba140c10cadcf288d494605cc6#diff-a988587cdea159026a58cc36592ce812

I can work on a mechanism to strip it out if it should not be included.

Reply:
Okay, I've added a kustomize patch to strip namespace from the redis-ha manifest.